src/amo/components/CollectionList/index.js

```diff
@@ -0,0 +1,174 @@
+/* @flow */
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import {
+  expandCollections, fetchUserCollections,
+} from 'amo/reducers/collections';
+import { getCurrentUser } from 'amo/reducers/users';
+import AuthenticateButton from 'core/components/AuthenticateButton';
+import { withFixedErrorHandler } from 'core/errorHandler';
+import translate from 'core/i18n/translate';
+import Button from 'ui/components/Button';
+import Card from 'ui/components/Card';
+import type {
+  CollectionsState, CollectionType,
+} from 'amo/reducers/collections';
+import type { UsersStateType } from 'amo/reducers/users';
+import type { ErrorHandlerType } from 'core/errorHandler';
+import type { I18nType } from 'core/types/i18n';
+import type { DispatchFunc } from 'core/types/redux';
+import type { ReactRouterLocation } from 'core/types/router';
+
+import './styles.scss';
+
+
+export type Props = {|
+  collections: Array<CollectionType> | null,
+  currentUsername: string | null,
+  dispatch: DispatchFunc,
+  errorHandler: ErrorHandlerType,
+  i18n: I18nType,
+  isLoggedIn: boolean,
+  loadingUserCollections: boolean,
+  location: ReactRouterLocation,
+|};
+
+export class CollectionListBase extends React.Component<Props> {
+  static defaultProps = {
+    creating: false,
+  };
+
+  componentDidMount() {
+    const {
+      collections,
+      currentUsername,
+      dispatch,
+      errorHandler,
+      loadingUserCollections,
+    } = this.props;
+
+    if (currentUsername && !loadingUserCollections && !collections) {
+      dispatch(fetchUserCollections({
+        errorHandlerId: errorHandler.id, username: currentUsername,
+      }));
+    }
+  }
+
+  render() {
+    const { i18n, isLoggedIn, location, collections } = this.props;
+    const myCollectionsHeader = i18n.gettext('My Collections');
```

Comment:
I know that this complies with the mock-ups but it looks weird to have the same title for the two cards that are displayed next to each other.

Reply:
The minimum would be to name the left card (in LTR mode) "About collections".